ui/src/app/views/workspace-edit/component.tsx

```diff
@@ -0,0 +1,584 @@
+import {ReactWrapperBase, withCurrentWorkspace, withRouteConfigData} from 'app/utils';
+import {navigate, userProfileStore} from 'app/utils/navigation';
+import {WorkspaceUnderservedPopulation} from 'app/views/workspace-edit-underserved-population/component';
+import {CdrVersion, DataAccessLevel, Workspace} from 'generated/fetch';
+import * as fp from 'lodash/fp';
+import * as React from 'react';
+
+export const ResearchPurposeItems = {
+  diseaseFocusedResearch: {
+    shortDescription: 'Disease-focused research',
+    longDescription: 'The primary purpose of the research is to learn more about a particular \
+    disease or disorder (for example, type 2 diabetes), a trait (for example, blood pressure), \
+    or a set of related conditions (for example, autoimmune diseases, psychiatric disorders).'
+  },
+  methodsDevelopment: {
+    shortDescription: 'Methods development/validation study',
+    longDescription: 'The primary purpose of the research is to develop and/or validate new \
+    methods/tools for analyzing or interpreting data (for example, developing more powerful \
+    methods to detect epistatic, gene-environment, or other types of complex interactions in \
+    genome-wide association studies). Data will be used for developing and/or validating new \
+    methods.'
+  },
+  aggregateAnalysis: {
+    shortDescription: 'Aggregate analysis to understand variation in general population',
+    longDescription: 'The primary purpose of the research is to understand variation in the \
+    general population (for example, genetic substructure of a population).'
+  },
+  controlSet: {
+    shortDescription: 'Control set',
+    longDescription: 'All of Us data will be used to increase the number of controls \
+    available for a comparison group (for example, a case-control study) to another \
+    dataset.'
+  },
+  ancestry: {
+    shortDescription: 'Population origins or ancestry',
+    longDescription: 'The primary purpose of the research is to study the ancestry \
+    or origins of a specific population.'
+  },
+  population: {
+    shortDescription: 'Restricted to a specific population',
+    longDescription: 'This research will focus on a specific population group. \
+    For example: a specific gender, age group or ethnic group.'
+  },
+  commercialPurpose: {
+    shortDescription: 'Commercial purpose/entity',
+    longDescription: 'The study is conducted by a for-profit entity and/or in \
+    support of a commercial activity.'
+  },
+  containsUnderservedPopulation: {
+    shortDescription: 'Focus on an underserved population',
+    longDescription: 'This research will focus on, or include findings on, distinguishing \
+    characteristics related to one or more underserved populations'
+  },
+  requestReview: {
+    shortDescription: 'Request a review of your research purpose'
+  }
+};
+
+export const toolTipText = {
+  header: 'A Workspace is your place to store and analyze data for a specific project.Each ' +
```

Comment:
FYI, backtick quotes allow multiline strings, which can make long blocks of text much nicer to edit.